apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -295,13 +295,15 @@ export const getCurrentQuestion = state => {
       ? assessmentQuestions[state.sectionAssessments.questionIndex]
       : null;
     if (question) {
+      const answers =
+        question.type === QuestionType.MULTI &&
+        (question.answers || []).map((answer, index) => {
+          return {...answer, letter: ANSWER_LETTERS[index]};
+        });
```

Comment:
this was breaking a UI test, so i fixed while i was here. `question.answers` was undefined, so `.map` was causing an error that broke rendering the Assessments tab.